akka-remote/src/main/scala/akka/remote/artery/AeronSource.scala

```diff
@@ -71,14 +71,17 @@ private[remote] object AeronSource {
 /**
  * INTERNAL API
  * @param channel eg. "aeron:udp?endpoint=localhost:40123"
+ * @param spinning the amount of busy spinning to be done synchronously before deferring to the TaskRunner
+ *                 when waiting for data
  */
 private[remote] class AeronSource(
   channel:        String,
   streamId:       Int,
   aeron:          Aeron,
   taskRunner:     TaskRunner,
   pool:           EnvelopeBufferPool,
-  flightRecorder: EventSink)
+  flightRecorder: EventSink,
+  spinning:       Int                = 0)
```

Comment:
for internal api I think it's better without default value to not accidentally forget about it